src/server/webhook-events.ts

```diff
@@ -66,3 +114,15 @@ export async function handlePullRequestReview(
         `Unexpected Pull Request Review Action: ${hookData.action}`);
   }
 }
+
+const statusToPRQuery = gql`
+query StatusToPR($queryString: String!) {
+  search(type: ISSUE, query:$queryString, first: 1) {
+    nodes {
+      ...on PullRequest {
+        id, number, title
```

Comment:
nit: can we newline separate these please? That's the style we've been using elsewhere.